addon/services/rollbar.js

```diff
@@ -14,7 +15,7 @@ export default Ember.Service.extend({
   }),
 
   rollbarClient(customConfig = {}) {
-    let config = Ember.assign(this.get('config'), customConfig);
+    let config = deepMerge(this.get('config'), customConfig);
```

Comment:
One noticeable issue.`{}`
`deepMerge({}, this.get('config'), customConfig);`

In the current case, we will just override the config which is not the expected behavior :/

Reply:
I've pushed the fix.

Reply:
Whoa! So fast! 🐎